Include/internal/pycore_optimizer.h

```diff
@@ -31,7 +31,8 @@ typedef struct {
     uint8_t oparg;
     uint16_t valid:1;
     uint16_t linked:1;
-    uint16_t chain_depth:14;  // Must be big engough for MAX_CHAIN_DEPTH - 1.
+    uint16_t was_run:1;
```

Comment:
Using a single bit means that setting this will take a read, mask and write instead of a just a write
Maybe reorganize this as:
```
    uint8_t valid:1;
    uint8_t linked:1;
    uint8_t chain_depth:6; // Must be big enough for MAX_CHAIN_DEPTH - 1.
    bool was_run;
```

Reply:
Good idea. And just to bikeshed the name, I like a name like `warm` a bit better than `was_run`. `was_run` makes me think that this indicates whether the executor has *ever* been run, which isn't true.